Formula/bento4.rb

```diff
@@ -20,40 +20,23 @@ class Bento4 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "be5b120e691952f835be698e3a271e719da3da0a825623a98b7ffdb3940ff884"
   end
 
-  depends_on xcode: :build
-  # artifact does not produce arm64 native binaries
-  depends_on arch: :x86_64
+  depends_on "cmake" => :build
   depends_on "python@3.10"
 
-  on_linux do
-    depends_on "cmake" => :build
-  end
-
   conflicts_with "gpac", because: "both install `mp42ts` binaries"
   conflicts_with "mp4v2", because: "both install `mp4extract` and `mp4info` binaries"
 
-  def install
-    if OS.mac?
-      cd "Build/Targets/universal-apple-macosx" do
-        xcodebuild "-target", "All", "-configuration", "Release", "SYMROOT=build"
-        programs = Dir["build/Release/*"].select do |f|
-          next if f.end_with? ".dylib"
-          next if f.end_with? "Test"
+  # Add support for cmake install.
+  # TODO: Remove in the next release.
+  patch do
+    url "https://github.com/axiomatic-systems/Bento4/commit/ba95f55c495c4c34c75a95de843acfa00f6afe24.patch?full_index=1"
+    sha256 "ba5984a122fd3971b40f74f1bb5942c34eeafb98641c32649bbdf5fe574256c5"
+  end
 
-          File.file?(f) && File.executable?(f)
-        end
-        bin.install programs
-      end
-    else
-      mkdir "cmakebuild" do
-        system "cmake", "..", *std_cmake_args
-        system "make"
-        programs = Dir["./*"].select do |f|
-          File.file?(f) && File.executable?(f)
-        end
-        bin.install programs
-      end
-    end
+  def install
+    system "cmake", "-S", ".", "-B", "cmakebuild", *std_cmake_args
+    system "cmake", "--build", "cmakebuild"
+    system "cmake", "--install", "cmakebuild"
```

Comment:
Probably worth rebuilding bottles to get headers, libraries, and cmake files. Not sure if worth adding revision bump as everything else should be the same and no one had complained on them missing

Reply:
It add significant features so a revision bump seems in order.